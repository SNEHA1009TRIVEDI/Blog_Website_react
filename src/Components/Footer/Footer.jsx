import React from 'react';
import './Footer.css';

const Footer = () => {
  return (
    <div className='body-padding footer'>
      &copy; Sneha Trivedi 2023
    </div>
  );
};

export default Footer;