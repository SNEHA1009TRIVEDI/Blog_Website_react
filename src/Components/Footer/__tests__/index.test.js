import { render } from "@testing-library/react";
import React from "react";
import Footer from "../Footer.jsx";

describe("Footer", () => {
  it("should renders without crashing ", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
