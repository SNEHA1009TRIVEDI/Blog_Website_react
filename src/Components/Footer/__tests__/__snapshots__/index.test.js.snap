// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should renders without crashing  1`] = `
<DocumentFragment>
  <div
    class="body-padding footer"
  >
    Â© Sneha Trivedi 2023
  </div>
</DocumentFragment>
`;
